From ce3553000066f327e452c71bf12ef14b0cfeba7b Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Mon, 24 Mar 2025 15:23:51 +0100
Subject: build: Add gkm_marshal header to libgkm sources

Otherwise the gkm_marshal header is sometimes not generated
while building libgkm

diff --git a/pkcs11/gkm/meson.build b/pkcs11/gkm/meson.build
index 9a3ab06c..23b57757 100644
--- a/pkcs11/gkm/meson.build
+++ b/pkcs11/gkm/meson.build
@@ -68,8 +68,9 @@ libgkm_dep = declare_dependency(
   link_with: libgkm,
   include_directories: [
     include_directories('.'),
-    include_directories('..'), # for gkm-marshal.h
-  ]
+    include_directories('..'), # for gkm/gkm-{crypto,assertion}.h
+  ],
+  sources: gkm_marshal[1],
 )
 
 # Tests
